### PR TITLE
MoheshwarAmarnathBiswas.YouTubeVideoDownloader version 1.2.1

### PR DESCRIPTION
**Pull request by fluentmoheshwar**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80070)

### CODE REVIEW

**Comment by fluentmoheshwar** [comment]
I don't know why this happening again and again
It works on my machine.
Maybe a bug of CI?

File: manifests/m/MoheshwarAmarnathBiswas/YouTubeVideoDownloader/1.2.1/MoheshwarAmarnathBiswas.YouTubeVideoDownloader.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+PackageVersion: 1.2.1
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/fluentmoheshwar/yt-download-py/releases/download/v1.2.1/YouTube_Video_Downloader_Installer.exe
+  InstallerSha256: 1F6E40B130D99EC10CEBA19C7CC1C6A61AE89FE869446D6B54D5830DE91F0120
+  InstallerSwitches:
+    Silent: /VERYSILENT
+    SilentWithProgress: /VERYSILENT
+    Upgrade: /VERYSILENT
```
Comment (Trenly) [inline manifests/m/MoheshwarAmarnathBiswas/YouTubeVideoDownloader/1.2.1/MoheshwarAmarnathBiswas.YouTubeVideoDownloader.installer.yaml:14]:
```suggestion
  InstallerSwitches:
    Custom: /ALLUSERS
  Scope: machine
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/fluentmoheshwar/yt-download-py/releases/download/v1.2.1/YouTube_Video_Downloader_Installer.exe
  InstallerSha256: 1F6E40B130D99EC10CEBA19C7CC1C6A61AE89FE869446D6B54D5830DE91F0120
  InstallerSwitches:
    Custom: /CURRENTUSER
  Scope: machine
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by fluentmoheshwar** [comment]
> image

It didn't appear on my PC though. Anyway, I changed as you said. is there any way to prevent this in future?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**